byzcoin/bcadmin/test.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-DBG_TEST=1
+DBG_TEST=2
```

Comment:
Put this back to 1.

Also, there's no new integration test for qrcode. There should be.

Reply:
Fixed for DBG_TEST, forgot it once more...

Put a test for the QR Code, but the tester will still have do scan the QR Code manually